onnxruntime/contrib_ops/cuda/math/complex_mul.cc

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "complex_mul.h"
+#include "complex_mul_impl.h"
+
+namespace onnxruntime {
+namespace contrib {
+namespace cuda {
+#define REGISTER_KERNEL_TYPED(T)                                  \
+  ONNX_OPERATOR_TYPED_KERNEL_EX(                                  \
+      ComplexMul,                                                 \
+      kMSDomain,                                                  \
+      1,                                                          \
+      T,                                                          \
+      kCudaExecutionProvider,                                     \
+      KernelDefBuilder()                                          \
+          .TypeConstraint("T", DataTypeImpl::GetTensorType<T>()), \
+      ComplexMul<T>);                                             \
+  ONNX_OPERATOR_TYPED_KERNEL_EX(                                  \
+      ComplexMulConj,                                             \
+      kMSDomain,                                                  \
+      1,                                                          \
+      T,                                                          \
+      kCudaExecutionProvider,                                     \
+      KernelDefBuilder()                                          \
+          .TypeConstraint("T", DataTypeImpl::GetTensorType<T>()), \
+      ComplexMulConj<T>);
+
+REGISTER_KERNEL_TYPED(float)
+REGISTER_KERNEL_TYPED(double)
```

Comment:
>double [](start = 22, length = 6)

Is double required? didn't see it in BuildKernelCreateInfo

Reply:
removed